tux/cogs/services/bookmarks.py

```diff
@@ -0,0 +1,39 @@
+import discord
+from discord.ext import commands
+from loguru import logger
+
+from tux.utils.embeds import EmbedCreator
+
+
+class Bookmarks(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_reaction_add(self, reaction: discord.Reaction, user: discord.User) -> None:
```

Comment:
should be [`on_raw_reaction_add`](https://discordpy.readthedocs.io/en/stable/api.html?highlight=on_reaction_add#discord.on_raw_reaction_add) instead. this change wont get you the user that reacted, you should fetch it manually using [`RawReactionActionEvent.user_id`](https://discordpy.readthedocs.io/en/stable/api.html?highlight=on_reaction_add#discord.RawReactionActionEvent.user_id)